src/Simple.OData.Client.Core/Simple.OData.Client.Core.csproj

```diff
@@ -5,6 +5,9 @@
     <RootNamespace>Simple.OData.Client</RootNamespace>
     <OutputPath>$(SolutionDir)/build/$(Configuration)/$(AssemblyName)/$(TargetFramework)/</OutputPath>
     <OutDir>$(OutputPath)</OutDir>
+    <SignAssembly>false</SignAssembly>
+    <DelaySign>false</DelaySign>
+    <AssemblyOriginatorKeyFile>C:\Projects\Simple.OData.Client\Key.snk</AssemblyOriginatorKeyFile>
```

Comment:
Revert these changes.

Reply:
Will do. Was in discussion with Vagif about pros&cons with signing. In my particular scenario I had to sign the assemblies and obviously my cleanup was not that thurough...